cvat-ui/src/components/analytics-page/analytics-page.tsx

```diff
@@ -42,321 +43,218 @@ function handleTimePeriod(interval: DateIntervals): [string, string] {
-                                    <Text>Quality</Text>
-                                </span>
-                            )}
-                            key='quality'
-                        >
-                            <TaskQualityComponent task={instance} onJobUpdate={onJobUpdate} />
-                        </Tabs.TabPane>
-                    </Tabs>
-                );
-                break;
-            }
-            case 'job':
-            {
-                backNavigation = (
-                    <Col span={22} xl={18} xxl={14} className='cvat-task-top-bar'>
-                        <GoBackButton />
-                    </Col>
-                );
-                title = (
-                    <Col className='cvat-job-analytics-title'>
-                        <Title level={4} className='cvat-text-color'>
-                            Analytics for
-                            {' '}
-                            <Link to={`/tasks/${instance.taskId}/jobs/${instance.id}`}>{`Job #${instance.id}`}</Link>
-                        </Title>
-                    </Col>
-                );
-                tabs = (
-                    <Tabs type='card'>
-                        <Tabs.TabPane
-                            tab={(
-                                <span>
-                                    <Text>Performance</Text>
-                                </span>
-                            )}
-                            key='overview'
-                        >
-                            <AnalyticsOverview
-                                report={analyticsReportInstance}
-                                onTimePeriodChange={onAnalyticsTimePeriodChange}
-                            />
-                        </Tabs.TabPane>
-                    </Tabs>
-                );
-                break;
-            }
-            default: {
-                throw new Error(`Unsupported instance type ${instanceType}`);
-            }
+    if (instanceType && instance) {
+        backNavigation = (
+            <Col span={22} xl={18} xxl={14} className='cvat-task-top-bar'>
+                <GoBackButton />
+            </Col>
+        );
+
+        let analyticsFor: JSX.Element | null = <Link to={`/projects/${instance.id}`}>{`Project #${instance.id}`}</Link>;
+        if (instanceType === 'task') {
+            analyticsFor = <Link to={`/tasks/${instance.id}`}>{`Task #${instance.id}`}</Link>;
+        } else if (instanceType === 'job') {
+            analyticsFor = <Link to={`/tasks/${instance.taskId}/jobs/${instance.id}`}>{`Job #${instance.id}`}</Link>;
         }
+        title = (
+            <Col>
+                <Title level={4} className='cvat-text-color'>
+                    Analytics for
+                    {' '}
+                    {analyticsFor}
+                </Title>
+            </Col>
+        );
+
+        tabs = (
+            <Tabs
+                type='card'
+                activeKey={activeTab || 'overview'}
```

Comment:
I believe we should add `'overview'` to useState as default value if `window.location.hash` is empty